pypeit/metadata.py

```diff
@@ -213,7 +260,14 @@ def _build(self, file_list, strict=True):
         if numfiles == 0:
             msgs.error("The headers could not be read from the input data files."
                        + msgs.newline() + "Please check that the settings file matches the data.")
-        
+
+#        # Add the index column
+#        data['index'] = np.arange(numfiles).tolist()
+
+        # Add the background_index column
+        #if bkg_pairs is not None:
+        self._add_bkg_pairs(data, bkg_pairs)
```

Comment:
So we're always going to add these columns?  For the moment, this means the `bkg_pairs` argument is meaningless, but we might bring it back with some automated method of setting the `comb` and `bkg` ids.

Reply:
yes, these are now always present.

so the `-bkg_pairs` argument now has the only
effect of printing them to the PypeIt file.

Reply:
Ah, okay, so it is still useful.  Makes sense.